static/templates/settings/organization_settings_admin.hbs

```diff
@@ -1,6 +1,24 @@
 <div id="organization-settings" data-name="organization-settings" class="settings-section">
     <form class="form-horizontal admin-realm-form org-settings-form">
 
+        <div id="org-pub-listing" class="org-subsection-parent">
+            <div class="subsection-header">
+                <h3>{{t "Public listing" }}</h3>
+                {{> settings_save_discard_widget section_name="pub-listing" }}
+            </div>
+            <div class="inline-block organization-settings-parent">
+                <div class="input-group">
+                    <label for="oss_showcase_policy" class="dropdown-title">{{t "List organization in community discovery page" }}
+                        <i class="fa fa-info-circle settings-info-icon realm_oss_showcase_policy_tooltip" data-toggle="tooltip"
+                          aria-hidden="true" title="{{t 'Only sponsored organizations can be listed publically.' }}"></i>
```

Comment:
Rather than using a title attribute, I'd like to use the `help_settings_link` widget to link a /help/ page /help/advertising-your-community.  For now the page can be a stub; I'll provide initial content before merging.